velox/expression/CastExpr-inl.h

```diff
@@ -301,6 +304,78 @@ VectorPtr CastExpr::applyDecimalToBooleanCast(
   return result;
 }
 
+template <typename FromNativeType>
+VectorPtr CastExpr::applyDecimalToVarcharCast(
+    const SelectivityVector& rows,
+    const BaseVector& input,
+    exec::EvalCtx& context,
+    const TypePtr& fromType) {
+  VectorPtr result;
+  context.ensureWritable(rows, VARCHAR(), result);
+  (*result).clearNulls(rows);
+  const auto simpleInput = input.as<SimpleVector<FromNativeType>>();
+  int precision = getDecimalPrecisionScale(*fromType).first;
```

Comment:
Would auto[precision, scale] = getDecimalPrecisionScale(*fromType) work?

Reply:
This caused an error in macos CI due to `scale` is accessed in a lambda-expression, so I changed to current way. Please check https://github.com/facebookincubator/velox/pull/6210/files#r1311282259.